### PR TITLE
Some App Distribution javadoc updates

### PR DESCRIPTION
**Pull request by lfkellogg**
Based on feedback from @rachelsaunders.

### CODE REVIEW

File: firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java [hunk 1]
```diff
@@ -191,28 +191,29 @@ public interface FirebaseAppDistribution {
    * Displays a notification that, when tapped, will take a screenshot of the current activity, then
    * start a new activity to collect and submit feedback from the tester along with the screenshot.
    *
-   * <p>On Android 13 and above, this method requires the <a
-   * href="https://developer.android.com/develop/ui/views/notifications/notification-permission">runtime
-   * permission for sending notifications</a>: {@code POST_NOTIFICATIONS}. If your app targets
-   * Android 13 (API level 33) or above, you should <a
+   * <p>On Android 13 and above, this method requires the runtime permission for sending
+   * notifications: <a
+   * href="https://developer.android.com/develop/ui/views/notifications/notification-permission">{@code
```
Comment (kaibolay) [inline firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java:196]: I'm not sure if `{@code …}` inside `<a>…</a>` gets rendered properly. Consider using `<pre>…</pre>` instead.

File: firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java [hunk 2]
```diff
@@ -221,32 +222,36 @@ void showFeedbackNotification(
...
```
Comment (kaibolay) [inline firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java:227]: I'm not sure if `{@code …}` inside `<a>…</a>` gets rendered properly. Consider using `<pre>…</pre>` instead.

**Review by kaibolay — APPROVED** [review]

File: firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java (hunk 1 above)
Comment (lfkellogg) [reply firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java:196]: Ah, good question but from the Javadoc preview it looks like it correctly inserts a `<code>...</code>` which should work.

File: firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java (hunk 2 above)
Comment (lfkellogg) [reply firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java:227]: Ditto

**Pull request merged**